addons/point_of_sale/models/pos_order.py

```diff
@@ -66,10 +66,18 @@ def _get_valid_session(self, order):
                         closed_session.id,
                         order['name'],
                         order['amount_total'])
+        rescue_session = PosSession.search([
+            ('name', '=like', '%(RESCUE FOR %(session)s)' % {'session': closed_session.name}),
```

Comment:
Ah right, I missed that in master the 'config_id' was added in the search. Right well, it doesn't matter much I suppose. If we do it this way there will be a difference in behavior in 10.0 and master when there are multiple rescue sessions open for the same pos.config.

I prefer to take master as an example of how it should be (with the ```rescue``` field):

```python
('state', 'not in', ('closed', 'closing_control')),
('rescue', '=', True),
('config_id', '=', closed_session.config_id.id),
```

which for 10.0 would become something like:

```python
('state', 'not in', ('closed', 'closing_control')),
('name', 'like', '(RESCUE FOR '), # fake rescue field
('config_id', '=', closed_session.config_id.id),
```

I'm not sure what the advantage is of adding additional divergence between both versions but anyway, you guys decide. I don't think there is a major downside to both approaches.